examples/governance/apps/src/bin/publisher.rs

```diff
@@ -0,0 +1,175 @@
+// Copyright 2024 RISC Zero, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This application demonstrates how to send an off-chain proof request
+// to the Bonsai proving service and publish the received proofs directly
+// to your deployed app contract.
+
+use alloy_sol_types::{sol, SolInterface};
+use anyhow::{Context, Result};
+use clap::Parser;
+use ethers::prelude::*;
```

Comment:
One suggestion I have is to drop `ethers` and instead use `alloy`. We need to do the same in the foundry-template as well. For now you could do something similar as in https://github.com/risc0/risc0-ethereum/blob/main/examples/erc20-counter/apps/src/bin/publisher.rs

Reply:
thank you for noticing that, I will definitely do that